### PR TITLE
Config

### PR DESCRIPTION
**Pull request by evanH13**
I still need to incorporate error-handling. I can also write more code that will update instead of overwriting values if args are passed via the CLI.

### CODE REVIEW

**Comment by wtanner** [comment]
Tests are failing. I'll see if I can fix it up.

File: spotifyutils/config.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    and replace with passed args. The auth modules are then used to generate access and refresh tokens. If additional args are NOT passed, read the configfile and
+    refresh access token if a refresh token exists. If a refresh token DOES NOT exist, use auth modules to generate access and refresh tokens.
+    If no arg is passed with the config method, check if a configfile exists in the default location. If a configfile DOES exist and a refresh toekn DOES exist
+    refresh the access token. If a configfile DOES exist and a refresh token DOES NOT exist, use the auth modules to generate refresh and access tokens. If a 
+    configfile DOES NOT exist in the default location, create config params based on passed args or user input. Use the auth modules to generate refresh and
+    access tokens. Finally, write all config params including refresh and access tokens to configfile. """
+
+    config = configparser.ConfigParser()
+
+    if kwargs['configfile']:
```
Comment (wtanner) [inline spotifyutils/config.py]: This fails if there is no configuration file specified. That is not expected behavior.

File: spotifyutils/config.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        configfile = kwargs['configfile']
+        if kwargs['client_id'] or kwargs['client_secret'] or kwargs['redirect_uri']:
```
Comment (wtanner) [inline spotifyutils/config.py]: Same with these.

**Comment by evanH13** [comment]
Sounds good

**Comment by wtanner** [comment]
> Sounds good
> 
> 

I made a pull request against this one for you to review, instead of a bunch of comments. When you are done reviewing we can merge that one into this one and then config is done.

**Review by wtanner — APPROVED** [review]

**Pull request merged**